crates/bevy_asset/src/server/mod.rs

```diff
@@ -1479,6 +1480,44 @@ impl AssetServer {
             }
         }
     }
+
+    /// Writes the default meta file for the provided `path`.
+    ///
+    /// Note if there is already a meta file for `path`, this function returns
+    /// `Err(WriteDefaultMetaError::MetaAlreadyExists)`.
+    pub async fn write_default_meta_file_for_path(
```

Comment:
I do like this general path, but using this method to generate the meta files for a project would ignore the "default processor configuration", as it just writes the default loader meta.

I'm cool with this method as-is, but I think we need an equivalent on the AssetProcessor as well that looks for a default processor configuration first and uses that _and then_ falls back to the logic here (aka the same logic that the processor uses currently when finding what meta to use at runtime).

Ultimately I think we'll want something that orchestrates the creation of meta files for a project for a given set of filters + configurations, but we can build that if and when we decide its needed.

Reply:
Done. Renamed this function to `write_default_loader_meta_file_for_path` and wrote a new version `AssetProcessor::write_default_meta_file_for_path`.